package.json

```diff
@@ -219,6 +219,11 @@
                     "default": true,
                     "description": "If one root workspace folder is nested in another root folder, look for the Rust config in the outermost root."
                 },
+                "rust-client.enableMultiProjectSetup": {
```

Comment:
I don't know much about the extension, didn't have the time to clone it yet, but wouldn't it be possible to make the extension work like :

- Normal cargo.toml at root of the project is tried
- If it fails the extension tries to determine if it's not a monorepo

I'd like your opinion on this, if it is quite some work I'd happily lend you a hand :slightly_smiling_face:

Reply:
> Normal cargo.toml at root of the project is tried

With this change the cargo.toml at root should also just work. 

>If it fails the extension tries to determine if it's not a monorepo
How would you detect, that I could imagine some wtf moments where you add a folder and all of the sudden the tooling stops working. I would rather make configuration for stuff like that.

> I'd like your opinion on this, if it is quite some work I'd happily lend you a hand

In general I would estimate that 99% will just always have it turned on.  So mostly this is about not breaking it for the 1%. 

and thx for the offer :)